dev/scripts/src/alluxio.org/build/cmd/flags.go

```diff
@@ -72,13 +80,15 @@ func parseTarballFlags(args []string) (*buildOpts, error) {
 	// profile specific flags
 	// all default values are set to empty strings to be able to check if the user provided any input, which would override the profile's corresponding predefined value
 	var flagProfile, flagTargetName, flagMvnArgs, flagLibModules, flagPluginModules string
+	var flagDisableTelemetry bool
 	cmd.StringVar(&flagProfile, "profile", defaultProfile, "Tarball profile to build; list available profiles with the profiles command")
 	cmd.StringVar(&flagMvnArgs, "mvnArgs", "", `Comma-separated list of additional Maven arguments to build with, e.g. -mvnArgs "-Pspark,-Dhadoop.version=2.2.0"`)
 	cmd.StringVar(&flagLibModules, "libModules", "",
 		fmt.Sprintf("Either a lib modules bundle name or a comma-separated list of lib modules to compile into the tarball; list available lib modules and lib module bundles with the plugins command"))
 	cmd.StringVar(&flagPluginModules, "pluginModules", "",
 		fmt.Sprintf("Either a plugin modules bundle name or a comma-separated list of plugin modules to compile into the tarball; list available plugin modules and plugin module bundles with the plugins command"))
 	cmd.StringVar(&flagTargetName, "target", "", "Name for the generated tarball; use '${VERSION}' as a placeholder for the version string")
+	cmd.BoolVar(&flagDisableTelemetry, "disableTelemetry", false, "Set true to disable Telemetry")
```

Comment:
can you add a TODO to eventually remove this? temporarily setting because it can't be toggled in configuration